### DISCUSSION TITLE
Pages Deployment Inconsistency

### DISCUSSION THREAD

**Discussion by borslap** [imfing/hextra#352] [General]
I am using a modified hextra template for a portfolio site with the default pages.yaml and a modified hugo.yaml - available at [github.com/borslap/borslap.github.io](https://github.com/borslap/borslap.github.io), but when deploying via github pages, the build directory keeps alternating between the parent directory (borslap.github.io) and the content subdirectory. 

So far, to overcome this problem, I've been modifying the baseURL in hugo.yaml to https://borslap.github.io or https://borslap.github.io/content, each time I got a Github Page Not Found Error, but this makes deployment unreliable. 

Any advice on how to fix the workflow would be appreciated!

**Comment by imfing** [comment]
I see no issue with your deployment. 
I think tweaking the github's workflow config is not necessary since it sets the base url properly.
